firstGame

Just-Den93 opened the pull request.

Just-Den93 committed:
firstGame

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+    'x': 0,
+    'o': 0,
+    'd': 0,
+};
```
empty line after block

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+    target.innerHTML = playerOne;
+    target.classList.add('playerOne');
+    count++;
+};
```
seperate functions with empty lines

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+    ];
+    
+    for(let i = 0; i < comb.length; i++) {
+
```
remove extra empty line

ivan629 replied:
move blocks[comb[i][0]].classList.contains('playerOne') &&
            blocks[comb[i][1]].classList.contains('playerOne') &&
            blocks[comb[i][2]].classList.contains('playerOne') to some util

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+                    blocks[comb[i][1]].classList.add('active');
+                    blocks[comb[i][2]].classList.add('active');
+                    res.innerText = 'Cross Win';
+                }, 100);
```
empty line after scope

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+                score.x += 1
+        }
+
+        else if (blocks[comb[i][0]].classList.contains('playerSecond') &&
```
move condition to util

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+                blocks[comb[i][1]].classList.add('active');
+                blocks[comb[i][2]].classList.add('active');
+                res.innerText = 'Zero Win';
+            }, 100);
```
seperate  functions invokes and settimeout

ivan629 commented on script.js:
```diff
@@ -0,0 +1,115 @@
+            score.o += 1
+        }
+
+        else if (count == 9 && i === 7) {
```
nove condition to util

Just-Den93 committed:
TIK TAK TOE
1. Functions isPlayerOneWin, isPlayerSecond, addActiveClass, addNegativeClass are deduced to util
2. Functions isPlayerOneWin, isPlayerSecond simplified with every();
3. Added handlePlayerClick function instead of the previous two;
